src/dropdown.option.ts

```diff
@@ -181,6 +179,32 @@ export default class DropdownOption extends LitElement {
   @property({ attribute: 'tabindex', reflect: true, type: Number })
   override readonly tabIndex = -1;
 
+  /**
+   * @default ''
+   */
+  @property({ reflect: true })
+  get value(): string {
```

Comment:
Moved it up. Previously it was below `render()`. The linter was apparently out to lunch that day.